packages/loader/src/ktx2/transcoder/BinomialLLCWorkerCode.ts

```diff
@@ -80,9 +80,15 @@ export function transcode(buffer: Uint8Array, targetFormat: any, KTX2File: any):
     PVRTC1_4_RGB = 8,
     PVRTC1_4_RGBA = 9,
     ASTC_4x4 = 10,
-    RGBA8 = 13
+    RGBA8 = 13,
+    BC6H = 22,
+    RGB_HALF = 24,
+    RGBA_HALF = 25
```

Comment:
`RGB_HALF` and `RGBA_HALF` never use?

Reply:
`RGBA_HALF` has used, `RGB_HALF` is enum of basisu, which has not been used in web.